common/Cargo.toml

```
@@ -6,41 +6,35 @@ edition = "2018"
 license = "GPL-3.0"
 
 [dependencies]
-codec = { package = "parity-scale-codec", version = "3.4.0", default-features = false, features = ["derive", "max-encoded-len"] }
-scale-info = { version = "2.3.1", default-features = false, features = ["derive"] }
-primitive-types = { version = "0.12.1", default-features = false, features = ["scale-info"] }
-log = "0.4.17"
-derive_more = "0.99.17"
+primitive-types = { workspace = true, features = ["scale-info"] }
+log = { workspace = true }
+derive_more = { workspace = true }
```

Isn't shorter to wite without brackets?
```suggestion
log.workspace = true
derive_more.workspace = true
```